### PR TITLE
release 1.18.0-snapshot.20210921.7889.1.1b473c2b

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
The `main` branch build for #10971 failed. #10979 should have fixed it, so let's try again.

This needs to be merged _after_ #10979.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Review by stefanobaghino-da — APPROVED** [review]
https://www.youtube.com/watch?v=dQw4w9WgXcQ

**Review by cocreature — APPROVED** [review]

**Comment by garyverhaegen-da** [comment]
So this is _also_ a bust, because (of course) the changes in #10979 need to be picked up, so this should target a commit after that rather than try to do the same commit as before. I'll make a new release PR after #10996 is merged (as we can't do both because git conflicts).

**Comment by garyverhaegen-da** [comment]
After discussion with the security team, we have decided that failed releases that went far enough to publish artifacts somewhere (this one published to mvn and npm) still count as standard changes.

**Pull request merged**
